tests/cypress/cypress/support/commands.js

```diff
@@ -75,54 +71,70 @@ Cypress.Commands.add('drupalLogout', () => {
  * @param {string} password
  *   The password for the user's account.
  */
-Cypress.Commands.add('loginAs', (username, password) => {
+Cypress.Commands.add("loginAs", (username, password) => {
   cy.drupalLogout();
-  cy.visit('/f64816be-34ca-4d5b-975a-687cb374ddf7');
-  cy.get('#edit-name')
-    .type(username);
-  cy.get('#edit-pass').type(password, {
+  cy.visit("/f64816be-34ca-4d5b-975a-687cb374ddf7");
+  cy.get("#edit-name").type(username);
+  cy.get("#edit-pass").type(password, {
     log: false,
   });
-  cy.get('.form-submit').contains('Log in').click();
+  cy.get(".form-submit").contains("Log in").click();
+});
+
+Cypress.Commands.add("verifyCallToActionBlock", (url, text, href) => {
+  // Custom command to verify the existence of a button within the "region-cta" block
+
+  // Visit the specified URL
+  cy.visit(url);
+
+  // cta section is tested
+  cy.get("#cta").should("exist");
+
+  //contained text within button
+  cy.contains(text);
```

Comment:
you should look for the text within the cta block -- something like `cy.get("#cta").contains(text) should do that